src/quill-editor.component.ts

```diff
@@ -97,6 +103,19 @@ export class QuillEditorComponent implements AfterViewInit, ControlValueAccessor
   onModelChange: Function = () => {};
   onModelTouched: Function = () => {};
 
```

Comment:
please move the inputs to the other inputs :)

Reply:
sure